docs/Using-Fleet/fleetctl-CLI.md

```diff
@@ -250,7 +250,7 @@ Now that the password has been reset, you will need to log in again using the up
 
 ### Create a new context for your standard user (optional)
 
-If you would like to use your API user by default for automated workflows and still use `fleetctl` with your standard user account, you can create an additional context. In the example below, a new `admin` context will be created:
+If you would like to use your API user by default for automated workflows and still use `fleetctl` with your standard user account, you can create an additional context. In the example below, a new `admin` context will be created.
```

Comment:
@Desmi-Dizney I would consider keeping the colon here based on what is standard in other documentation websites.  (It's referring to the code snippet below)

Reply:
Thanks for the info. I'm changing it back.